src/Device/Driver/AirControlDisplay.cpp

```diff
@@ -201,6 +202,19 @@ ACDDevice::OnSensorUpdate(const MoreData &basic)
   }
 }
 
+void
+ACDDevice::OnCalculatedUpdate(const MoreData &basic,[[maybe_unused]] const DerivedInfo &calculated)
+{
+   NullOperationEnvironment env;
+
+   if (basic.settings.qnh_available.IsValid()){
+	 char buffer[100];
```

Comment:
bad indent

Reply:
It seems @lordfolken already complained about this one, but you ignored his complaint.

Reply:
Fixed